packages/core/src/stateUtils.ts

```diff
@@ -931,15 +931,15 @@ export function getActions<TContext, TEvent extends EventObject>(
     flatten(
       Array.from(entryStates).map(entryNode => {
         return [
-          ...getActivities(entryNode).map(activity => start(activity)),
+          ...getInvocations(entryNode).map(invokeDef => start(invokeDef)),
```

Comment:
it would be better to convert those early `entryNode.invoke` could already be invoke-definitions to avoid converting them each time in `getActions`